sw/device/lib/base/mmio.h

```diff
@@ -119,6 +136,28 @@ inline void mmio_region_write32(mmio_region_t base, ptrdiff_t offset,
                                 uint32_t value) {
   ((volatile uint32_t *)base.base)[offset / sizeof(uint32_t)] = value;
 }
+#else   // MOCK_MMIO
+/**
+ * "Instrumented" reg32_t.
+ *
+ * Instead of containing a volatile pointer, reg32_t becomes a |void *| when
+ * |-DMOCK_MMIO| is enabled. This makes it incompatible with the non-mock
+ * version of |reg32_t|, which prevents users from being able to access the
+ * pointer inside.
```

Comment:
Both instances of `reg32_t` should be `mmio_region_t`.

Reply:
Done.